src/modules/pages/components/ExpenditurePage/constants.ts

```diff
@@ -11,6 +11,8 @@ import { newFundingSource } from '~dashboard/ExpenditurePage/Streaming/constants
 
 import { ExpenditureEndDateTypes, ExpenditureTypes } from './types';
 
+export const fixTriggerEventName = 'fix-trigger';
```

Comment:
This is better to have in snake case `FIX_TRIGGER_EVENT_NAME`, that's our default fro constants like this.